packages/epics/src/proposals/components/proposals-section.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+import { FC } from 'react';
+import { useProposals } from '../hooks/useProposals';
+import { ProposalTabs } from './proposal-tabs';
+import { ProposalsFilter } from './proposal-filter';
+import ProposalList from './proposal-list';
+
+type ProposalSectionProps = Record<string, never>
+
+export const ProposalsSection: FC<ProposalSectionProps> = () => {
+  const { proposalsCount, activeTab, setActiveTab, loadMore, 
+          filterProposalsByStatus, proposals } = useProposals();
+
+  const filteredProposals = filterProposalsByStatus(activeTab);
+
+  const isLoading = proposals.length === 0;
+
+  const renderContent = () => {
+    if (isLoading) {
+      return <div>Loading...</div>;
+    }
+
+    return (
+      <>
+        <ProposalsFilter 
+          value={activeTab} 
+          count={proposalsCount} 
+          onChange={setActiveTab} 
+        />
```

Comment:
I think a contextually more clear name would be something like: `setActiveFilter` or `setActiveStatus`